commands/cleanup.go

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/MichaelMure/git-bug/entities/bug"
+	"github.com/MichaelMure/git-bug/entities/identity"
+
+	"github.com/spf13/cobra"
+
+	"github.com/MichaelMure/git-bug/commands/execenv"
+)
+
+func newCleanUpCommand() *cobra.Command {
+	env := execenv.NewEnv()
+
+	cmd := &cobra.Command{
+		Use:     "cleanup",
+		Short:   "Cleanup git-bug from git repository",
+		Long:    "Cleanup git-bug from git repository",
+		PreRunE: execenv.LoadRepo(env),
+		RunE: execenv.CloseBackend(env, func(cmd *cobra.Command, args []string) error {
+			return runCleanup(env)
+		}),
+	}
+
+	return cmd
+}
+
+func runCleanup(env *execenv.Env) error {
+
+	fmt.Println("cleaning bug items..")
```

Comment:
"cleaning bug items.." --> "cleaning bugs ..."
Plus, you should use `env.Out`

Reply:
Changed text, using env.Out now.